packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -7001,11 +7000,11 @@ The Create object is composed of a single setting that determines whether the ap
 
 | Property         | Description                                                  | DataType                                                                    | Nullable | Unique | Readonly |
 | ---------------- | ------------------------------------------------------------ | --------------------------------------------------------------------------- | -------- | ------ | -------- |
-| lifecycleCreate            | setting that determines whether the updates to a user in Okta will be update a user in the application   | [Lifecycle Create Setting Object](##lifecycle-create-setting-object)      | TRUE    | FALSE   | FALSE    |           |
+| lifecycleCreate            | Setting that determines whether the updates to a user in Okta will be update a user in the application   | [Lifecycle Create Setting Object](##lifecycle-create-setting-object)      | TRUE    | FALSE   | FALSE    |           |
 
 ###### Update object
 
-The Create object is composed of multiple settings that determine whether the updates to a user in Okta will update a user in the application.
+The Create object is composed of multiple settings that determines whether an Okta user profile, user deactivation, or password change will update a user in the application.
```

Comment:
Either leave this as `determine` (because it is the multiple settings (so plural) that determine the result) or rewrite:
> There are multiple settings in the Create object that determine if an Okta user profile change, user deactivation, or a password change will update a user in the application.